packages/lib/src/index.ts

```diff
@@ -115,6 +112,12 @@ export interface CompletionOptions<TModel, TInput extends Input = Input> {
   numOutputs?: number
 }
 
+export function isCompletionOptions(
+  options: CompletionOptions<string, Input>
```

Comment:
options should be a broader type than the result of the type guard (currently it's the same type so this is kind of a no-op fn for typescript)

Reply:
refactored out to options type and added that here